src/arch/src/aarch64/gic/regs/mod.rs

```diff
@@ -5,11 +5,44 @@ mod redist_regs;
 use crate::aarch64::gic::{Error, Result};
 use kvm_bindings::kvm_device_attr;
 use kvm_ioctls::DeviceFd;
+use std::fmt::Debug;
 use std::iter::StepBy;
 use std::ops::Range;
 use versionize::{VersionMap, Versionize, VersionizeResult};
 use versionize_derive::Versionize;
 
+#[derive(Debug)]
+pub struct GicRegData<T: Versionize> {
```

Comment:
Abstraction looks good! For consistency with other components' states I would rename this to `GicRegState`

Reply:
Done